rainbow/src/main/java/org/projectrainbow/Updater.java

```diff
@@ -0,0 +1,61 @@
+package org.projectrainbow;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.net.UnknownHostException;
+
+import javax.net.ssl.SSLHandshakeException;
+import java.lang.NumberFormatException;
+
+import org.projectrainbow.launch.Bootstrap;
+
+/**
+ * Check if Rainbow is outdated.
+ * 
+ * @author Isaiah Patton
+ */
+public class Updater {
+    public static String checkForUpdate() {
+        String latest = Updater.getLatestVersion();
+        if (!latest.startsWith("ERROR")) {
+            if (Bootstrap.buildNumber == latest) {
```

Comment:
Still not working. This condition will **never** evaluate to true.

![2017-02-20_12 11 53](https://cloud.githubusercontent.com/assets/7014077/23122792/fdf54c2a-f765-11e6-999e-6b6f44be1fd9.png)